tests/test_cli.py

```diff
@@ -170,10 +170,15 @@ def test_construct_workflow_name():
     from reana.reana_dev.run import construct_workflow_name
 
     for (input_value, output_expected) in (
-        (("reana", "cwl"), "reana-cwl"),
-        (("reana-demo-root6-roofit", "yadage"), "root6-roofit-yadage"),
+        (("reana", "cwl", "kubernetes"), "reana-cwl-kubernetes"),
+        (
+            ("reana-demo-root6-roofit", "yadage", "htcondorcern"),
+            "root6-roofit-yadage-htcondorcern",
+        ),
```

Comment:
Note: some examples use different conventions, e.g. RECAST is only available in Yadage, while the matrix assumes that `reana.yaml` always means Serial. We may want to introduce a new config variable to catch these exceptional cases so that we don't generate `atlas-recast-serial-kubernetes` but rather the proper `atlas-recast-yadage-kubernetes` name.